ui/app/AppLayouts/Wallet/popups/simpleSend/SimpleSendModal.qml

```diff
@@ -327,8 +327,9 @@ StatusDialog {
                                                                     selectedCollectibleEntry.item.symbol: ""
 
         readonly property double maxSafeCryptoValue: {
+            root.assetsModel.ModelCount.count
```

Comment:
hmm I don't see assetsModel used anywhere in this calculation, do we have some other more direct property we can bind this to?

Reply:
hmm damn I just read this comment after merging, got excited with all green below!!

Good point, I did try and use the selectedAccountAddress but that didnt seem right, I tried with selectedAssetEntry as well, but it failed too, but maybe I need to think more, cause its a good point!